src/Mono.Android/metadata

```diff
@@ -1922,6 +1922,10 @@
   <attr api-since="35" path="/api/package[@name='android.media']/class[@name='MediaRouter2']/method[@name='getRouteListingPreference' and count(parameter)=0]" name="propertyName"></attr>
   <attr api-since="35" path="/api/package[@name='android.media']/class[@name='MediaRouter2']/method[@name='setRouteListingPreference' and count(parameter)=1 and parameter[1][@type='android.media.RouteListingPreference']]" name="propertyName"></attr>
 
+  <!-- These new default interface methods on a Listener interface cause duplicate named EventHandlers to be generated, change to a unique name. -->
+  <attr api-since="35" path="/api/package[@name='android.app']/interface[@name='AppOpsManager.OnOpActiveChangedListener']/method[@name='onOpActiveChanged' and count(parameter)=8]" name="managedName">OnOpActiveChanged2</attr>
```

Comment:
Also not sure I like `count(parameter)=8`, as if they add an 8-parameter overload in the future, it'll match.

"Safer" would likely to use `//method/@jni-signature`.

Reply:
Updated to use the "full" generated metadata path.